sentry_sdk/serializer.py

```diff
@@ -253,10 +277,18 @@ def _serialize_node_impl(
         if is_databag is None:
             is_databag = _is_databag()
 
-        if is_databag and remaining_depth is None:
-            remaining_depth = MAX_DATABAG_DEPTH
-        if is_databag and remaining_breadth is None:
-            remaining_breadth = MAX_DATABAG_BREADTH
+        if is_request_body is None:
+            is_request_body = _is_request_body()
```

Comment:
is this needed? not sure if it ever is true if it's already not true above?

Reply:
Oh yeah, thanks for spotting that